### ISSUE TITLE
I have Question on train

### ISSUE THREAD

**Issue by ssb0621** [ronghuaiyang/arcface-pytorch#52]
I train resnet34(pretrain=false) and i tried 1 epoch about own my dataset
when i run 1 epoch, All train loss is 0.0.

and i re-train my dataset and load my 1epoch pth(load_state_dict(my 1epoch model path)) then accuracy about my train is  up.
just 1 epoch, train loss is about 0.90

is it possible? i think it has something wrong.

Can anyone tell me why?
